static.json

```diff
@@ -1,6 +0,0 @@
-{
-  "root": "build/",
-  "routes": {
-    "/**": "index.html"
-  }
-}
```

Comment:
I don't know how this deletion ended up in this PR, neither how to "undo" it since it does not come from an actual commit — n.b. we'll have to revert this after merging